example_package/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "example_package"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+# This is what we need for using this package in our mono-repo
+roslibrust = { path = "../roslibrust" }
```

Comment:
You can pass both. Cargo will prefer paths if they're available over the version. Not sure if it's worth updating and adding a note about that here or not.

Reply:
Yeah will punt, I actually had a line with both in there, but figured that was actually more confusing as an example